buildpipeline/linux.arm64.groovy

```diff
@@ -14,32 +14,19 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-arm64-a
     }
 
     def logFolder = getLogFolder()
+    def commonprops = "/p:ArchGroup=${params.AGroup} /p:ConfigurationGroup=${params.CGroup}"
 
```

Comment:
This leg needs some more work. We need to generate the version.c file correctly as well as we need to do the managed build as x64 instead of arm64. @sdmaclea is the separation still necessary?

Reply:
Building managed code on arm64 is still unstable last I checked.  So it would still need to be cross built.